tests/test_pyrflx.py

```diff
@@ -48,26 +59,40 @@ def setUpClass(cls) -> None:
                 f"{cls.specdir}/tls_alert.rflx",
                 f"{cls.specdir}/icmp.rflx",
                 f"{cls.testdir}/test_odd_length.rflx",
+                f"{cls.specdir}/ipv4.rflx",
+                f"{cls.testdir}/array_message.rflx",
+                f"{cls.testdir}/array_type.rflx",
+                f"{cls.specdir}/udp.rflx",
+                f"{cls.specdir}/tlv.rflx",
             ]
         )
-        cls.package_tlv = pyrflx["TLV"]
+        # einzelne Packages
```

Comment:
Please translate or remove.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/198/commits/1ab7b6311e8bc2f82f25c3c4b96d70a135337223.